components/BottomNavigation/tests/unit/BottomNavigationItemViewTests.m

```diff
@@ -127,4 +132,40 @@ - (void)testSetSelectedItemTintColorUpdatesInkColor {
   XCTAssertNotEqualObjects(item2.inkView.inkColor, item2DefaultInkColor);
 }
 
+- (void)testBadgeCenterIsCorrectWithoutRTL {
+  // Given
+  MDCBottomNavigationItemView *itemView = [[MDCBottomNavigationItemView alloc] init];
+  itemView.iconImageView.frame = CGRectMake(8, 8, 24, 24);
+
+  // When
+  CGPoint badgePoint = [itemView badgeCenterFrom:CGRectStandardize(itemView.iconImageView.frame)
+                                           isRTL:NO];
+
+  // Then
+  CGRect iconFrame = itemView.iconImageView.frame;
+  CGPoint expectPoint =
+      CGPointMake(CGRectGetMaxX(iconFrame),
+                  CGRectGetMinY(iconFrame) + kMDCBottomNavigationItemViewBadgeYOffset);
+  XCTAssertEqualWithAccuracy(badgePoint.x, expectPoint.x, 0.001);
+  XCTAssertEqualWithAccuracy(badgePoint.y, expectPoint.y, 0.001);
+}
+
+- (void)testBadgeCenterIsCorrectWithRTL {
+  // Given
+  MDCBottomNavigationItemView *itemView = [[MDCBottomNavigationItemView alloc] init];
+  itemView.iconImageView.frame = CGRectMake(8, 8, 24, 24);
```

Comment:
Tip: Make the expected badge center point an explicitly-declared value at the top here so it's easy for maintainers to read the test:

```objectivec
  // Given
  MDCBottomNavigationItemView *itemView = [[MDCBottomNavigationItemView alloc] init];
  itemView.iconImageView.frame = CGRectMake(8, 8, 24, 24);
  CGPoint expectedCenter = CGPointMake(32, 12);

   // When
  CGPoint badgePoint = [itemView badgeCenterFrom:CGRectStandardize(itemView.iconImageView.frame)
                                           isRTL:NO];
   // Then
  XCTAssertEqualWithAccuracy(badgePoint.x, expectedCenter.x, 0.001);
  XCTAssertEqualWithAccuracy(badgePoint.y, expectedCenter.y, 0.001);
```